### DISCUSSION TITLE
How to Recieve Bounce Email

### DISCUSSION THREAD

**Discussion by absolutezack** [postalserver/postal#2615] [Help with using Postal]
Hello Guys,

I'm using Zimbra as a primary mail server and Postal as a relay server to handle email blasting.
Email blasting flow is like this: Zimbra >> Postal >> Recipient. 
Sometimes there are problems with email recipients, such as when their email quota is full or the email doesn't even exist. If this problem occurs, usually there will be a bounce email sent to the email sender. Is it possible for me to get that bounced email? 

Thanks

**Comment by willpower232** [comment]
You can configure the return path route here https://docs.postalserver.io/other/auto-responders-and-bounces

I don't know how zimbra works though so can't say whether it will fully work or not.
